snuba/replacers/replacer_processor.py

```diff
@@ -1,10 +1,26 @@
 from abc import ABC, abstractmethod
+from enum import Enum
 from typing import Any, Generic, Mapping, NamedTuple, Optional, TypeVar
 
 from snuba.datasets.events_processor_base import ReplacementType
 from snuba.datasets.schemas.tables import WritableTableSchema
 
 
+class ReplacerState(Enum):
+    EVENTS = "events"
+    ERRORS = "errors"
+
+
+class ReplacementMessageMetadata(NamedTuple):
```

Comment:
This should contain the ReplacerState as well.

I intend to have another consumer group running with a different ReplacerState for the new errors cluster. The existing parameters would overlap for both the consumer groups. So we need to be able to separate their processing logic.